### PR TITLE
Naming Scheme for namedParser

### PR DESCRIPTION
**Pull request by cchantep**
It would actually be really great to have a way of defining a NamingSchema to the namedParser.

Usecase: Mostly I encode my PostgreSQL as such:
> customer_id, invoice_mail

While on Scala I use the following schema:
> customerId, invoiceMail

### CODE REVIEW

**Comment by cchantep** [comment]
Ok for review

**Comment by schmitch** [comment]
LGTM (tested mostly the functionality)

File: build.sbt [hunk 1]
```diff
@@ -30,6 +30,9 @@ lazy val anorm = project
   .settings(
   scalacOptions += "-Xlog-free-terms",
     binaryIssueFilters ++= Seq(
+      missMeth("anorm.DefaultParameterValue.stringValue"/* deprecated */),
+      missMeth("anorm.ParameterValue.stringValue"/* deprecated */),
+      missMeth("anorm.BatchSql.apply"/* was deprecated */),
```
Comment (cchantep) [inline build.sbt]: @gmethvin Deprecated in 2.3.8

Comment (gmethvin) [inline build.sbt]: Okay, but that means this will require a 2.6 (or greater) version of anorm, right? What is anorm's policy about removing methods?

Comment (cchantep) [inline build.sbt]: For me that me it could have been removed since 2.4.0 as the deprecation was validated by a new major release between.

**Pull request merged**